Makefile

```diff
@@ -27,6 +27,10 @@ endif
 build:
 	cargo build --release
 
+.PHONY: install
+install:
+	cargo install --path .
```

Comment:
Forgive the persnicketing, but `make build` does a release build, while this does a debug build - should they be consistent?

Reply:
I believe `cargo install` uses the release profile by default (opposite to `build` which uses `dev`, i.e. `--debug`)

Reply:
Doh, you're right @fibonacci1729.  Apologies!